index.js

```diff
@@ -6,112 +6,147 @@ continueDisplayDog.style.display = "none";
-                let arrayOfImage = document.getElementById('petImage');
-                arrayOfImage.src = `https://cataas.com/${url}`
-                continueDisplayCat.style.display = 'block';
+if (btn !== null) {
+    btn.addEventListener('click', function () {
+        let cat = 0;
+        let dog = 0;
+        continueDisplayCat.style.display = "none"
+        continueDisplayDog.style.display = "none"
+        continueDisplayLlama.style.display = "none"
 
-            })
-    } else if (cat < dog) {
-        fetch(`https://random.dog/woof.json`)
-            .then(function (response) {
-                console.log(response);
-                return response.json();
-            }).then(function (dogData) {
-                dogAPIData = dogData
-                let url = dogAPIData.url
+        let selectedAnswers = document.querySelectorAll("input:checked") // creating an array with all the checked values in it
+        //evaluating question 1
 
-                let arrayOfImage = document.getElementById('petImage');
-                arrayOfImage.src = url
-                continueDisplayDog.style.display = 'block';
+        if (selectedAnswers[0].value === "low") {
+            cat++;
+        } else if (selectedAnswers[0].value === "medium") {
+            cat++;
+            dog++;
+        } else {
+            dog++;
+        };
+        //evaluating question 2
+        if (selectedAnswers[1].value === "never") {
+            dog++;
+        } else if (selectedAnswers[1].value === "sometimes") {
+            cat++;
+            dog++;
+        } else {
+            cat++;
+        }
+        // evaluating question 3
+        if (selectedAnswers[2].value === "yes") {
+            dog++;
+        } else if (selectedAnswers[2].value === "no") {
+            cat++;
+        }
+        //evaluating question 4
+        if (selectedAnswers[3].value === "0") {
+            cat++;
+            dog++;
+        } else if (selectedAnswers[3].value === "25") {
+            cat++;
+            dog++;
+        } else if (selectedAnswers[3].value === "50") {
+            cat++;
+            dog++;
+        } else if (selectedAnswers[3].value === "75") {
+            cat++;
+            dog++;
+        } else {
+            cat++;
+            dog++;
+        }
 
-            })
-    } else if (cat === dog) {
-        let arrayOfImage = document.getElementById('petImage')
-        arrayOfImage.src = `llama.jpg`;
-        continueDisplayLlama.style.display = 'block';
+        //evaluating question 5
+        if (selectedAnswers[4].value === "NoFur") {
+            cat++;
+        } else if (selectedAnswers[4].value === "LittleFur") {
+            cat++;
+            dog++;
+        } else {
+            dog++;
+        };
+
+        console.log("You are a cat person:", cat)
+        console.log("You are a dog person", dog)
+        // if certain answers are selected
+        // then display dog gif
+
+        // otherwise
+        //display cat gif
```

Comment:
Pseudo-Code seems to be implemented below, possibly remove this code?

Reply:
removed the pseudocode